examples/Manifest.toml

```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.7.3"
+julia_version = "1.7.1"
```

Comment:
@Zhengyu-Huang, it looks like you're using an older version of Julia, can you upgrade? Let's revert these manifest file changes